src/components/modal/_modal.scss

```diff
@@ -0,0 +1,57 @@
+.modal {
+    display: none; 
+    position: fixed; 
+    z-index: 200; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(31,31,31); 
+    background-color: rgba(31,31,31,0.75); 
+
+  
+    .modal-content {
```

Comment:
I know you previously stated your hesitance towards styling so I'll try to go easy on you here :)

We've been using BEM which—among other benefits—helps keep the styling flattened. As it is right now, these styles will render into the following CSS: `.modal .modal-content h2`. To update this approach you would just need to update the `.modal-content` class to `.modal__content` and we can use SASS nesting to simplify writing out these classes by doing the following:

```
.modal{
    &__content{
        //styles
    }
}
```

Which would compile to the following CSS: `.modal__content{//styles}`